src/transformers/integrations/deepspeed.py

```diff
@@ -414,7 +414,7 @@ def deepspeed_init(trainer, num_training_steps, inference=False):
     return optimizer, lr_scheduler
 
 
-def deepspeed_load_checkpoint(deepspeed_engine, checkpoint_path, load_module_strict=True):
+def deepspeed_load_checkpoint(deepspeed_engine, checkpoint_path, load_module_strict=False):
```

Comment:
can we somehow revert this and just force-set it to `True` in our trainer?

Reply:
That's a fair point! I'll push a change now